### DISCUSSION TITLE
Using KQL for search results

### DISCUSSION THREAD

**Discussion by kadejd** [microsoft-search/pnp-modern-search#2121] [Q&A]
Hi this might not be the best place to ask this question, but I was wondering if anyone could help,

Currently I populate the url with a search string to use within the query template in search results to match a "refinablestring" type and bring back results this works but I'm unsure how to

Use "or" within the query template so two refs to bring back more results

And also to be able to use a wild card so "c324#323" so that I can lookup multiple Ids matching one "refinable string"

I have been suggested to use KQL but I'm unsure how I would then use this to populate the search results

Hopefully this makes sense, is anyone able to assist, happy to supply more info!

**Comment by wobba** [comment]
Can you share what your input would be, and what you want the final query to look like sort of?
